content/markdown/concepts/footnotes/footnotes.md

```diff
@@ -0,0 +1,53 @@
+---
+Title: 'footnotes'
+Description: 'Footnotes are subscript numbers that appears at the end of a word or phrase. This is used in documentation to add notes and references in a way that does not clutter the files.'
+Subjects:
+  - 'Developer Tools'
+  - 'Web Development'
+Tags:
+  - 'Footnotes'
```

Comment:
Since this tag is not in Codecademy tags file you can add it there

Reply:
Hey @cigar-galaxy82. Can you elaborate on this please?

Reply:
By this, I mean we have a list of tags that are in the tags.md file and since this tag is not there already you can add it there